src/api/routes/controllers/goals.controller.js

```diff
@@ -123,3 +123,41 @@ exports.modifySubGoal = async (req, res, next) => {
     next(error);
   }
 };
+
+exports.addUser = async (req, res, next) => {
+  try {
+    const goalId = req.params.id;
+    const email = req.body.email;
```

Comment:
```suggestion
    const { email } = req.body;
```

이건 바로 구조분해할당해서 가져와도 될 것 같습니다~!

Reply:
해당 부분도 수정하도록 하겠습니다🙏